packages/core/test/merkle-distributor/MerkleDistributor.js

```diff
@@ -251,7 +251,7 @@ contract("MerkleDistributor.js", function(accounts) {
           },
           { from: rando }
         );
-        assertApproximate(87259, claimTx.receipt.gasUsed);
+        assertApproximate(87749, claimTx.receipt.gasUsed);
```

Comment:
gas went up slightly?

Reply:
ye, looks like it! some went down though.